routes/error-tracker.js

```diff
@@ -49,6 +49,47 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  let chromeStackTraceRegex = /\s*at (?:([^\n]*) )?([^\n]+):(\d+):(\d+)[^]/gm;
+  let mozillaSafariStackTraceRegex = /^([^@\n]*)@(.+):(\d+):(\d+)$/gm;
+  let validException = '';
+  let match;
+  let validExceptions = [];
+  if (chromeStackTraceRegex.test(exception)) {
+    // Reset the state of the regex to capture all matches.
+    chromeStackTraceRegex.lastIndex = 0;
+    while (match = chromeStackTraceRegex.exec(exception)) {
+      validExceptions.push(match[0]);
+    }
+    validException = validExceptions.join('');
+    // Remove first blank line
+    validException = validException.trim();
+  } else {
+    let otherMatch;
+    while (otherMatch = mozillaSafariStackTraceRegex.exec(exception)) {
+      validExceptions.push(safariOrMozillaToChrome(otherMatch[0]));
+    }
+    validException = validExceptions.join('\n');
+  }
+  return validException;
+}
+
+/**
+ * @param {string} exception
+ * @return {string} chromeLikeException
+ */
+function safariOrMozillaToChrome(exception) {
+  let context = exception.substring(0,
```

Comment:
This parsing is unnecessary, we've already extracted this information into the regex match:

```js
// match = ['t2@https://google.com/file.js:2:4', 't2', 'https://google.com/file.js:2:4'];
return `  at ${match[1]} ${match[2]}`;
```